lib/helpers/collapse-whitespace.js

```diff
@@ -1,7 +1,7 @@
 export default function collapseWhitespace(string) {
   return string
-    .replace(/[\t\r\n]/g, ' ')
-    .replace(/ +/g, ' ')
+    .replace(/\s+/g, ' ')
+    .replace(/\u200b+/g, '')
     .replace(/^ /, '')
```

Comment:
Is there a particular reason we don't use `.trim()` here? It's ES5 so pretty much [all browsers these days support it](http://kangax.github.io/compat-table/es5/#test-String_properties_and_methods_String.prototype.trim_a_href=_https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim_title=_MDN_documentation_img_src=_../mdn.png_alt=_MDN_(Mozilla_Development_Network)_logo_width=_15_height=_13_/_/a_nbsp;)